File: leetcode_solution/Reverse Integer.py
class Solution:
    # @return an integer
    def reverse(self, x):
        output = 0
        flag = False
        if x < 0:
            flag = True
            x = -x
        
        while x > 0:
            number = x - x/10 * 10
            output = output * 10 + number
            x = x/10
        
        if flag == True:
            output = -output
        return output
        
        

File: leetcode_java/Reverse Integer.java
public class Solution {
    public int reverse(int x) {
        Boolean Flag = FALSE;
        if(x == 0) {
        	return 0;
        }
        if(x < 0) {
        	x = -x;
        	Flag = TRUE;
        }

        int res = 0;
        while(x > 0) {
        	int left = x % 10;
        	x = x / 10;
        	res = res * 10 + left;
        }
        if(Flag == TRUE) {
        	res = -res;
        }
        return res;
    }
}